### PR TITLE
Introduce 'tagpr' to automate bumping semantic version

### PR DESCRIPTION
**Pull request by Rindrics**
related to #4

- this PR will introduce [`tagpr`](https://github.com/marketplace/actions/automate-pull-request-generation-and-tagging-for-releases-using-tagpr) to automate tagging (bumping semantic version)
    - it increments patch version on by default
    - it increments minor- or major- versions by adding the corresponding labels 
        - for minor: https://github.com/route06/aws-cost-cli/labels/minor
        - for major: https://github.com/route06/aws-cost-cli/labels/major
- automated 'releasing' will be enabled after closing https://github.com/route06/aws-cost-cli/issues/5

### CODE REVIEW

File: .github/workflows/cd.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# .github/workflows/cd.yml
```
Comment (bvlion) [inline .github/workflows/cd.yaml]: Is this comment necessary?
```suggestion
```

**Review by bvlion — APPROVED** [review]
I made one comment, but it doesn't affect the functionality, so I'm approving it.

File: .github/workflows/cd.yaml (hunk 1 above)
Comment (rindrics) [reply .github/workflows/cd.yaml]: no~~, thanks!!

Comment (rindrics) [reply .github/workflows/cd.yaml]: fixed: https://github.com/route06/aws-cost-cli/pull/13/commits/a8e9029d61489325ef1c19a2d016d8c0537ab47b

**Comment by rindrics** [comment]
thx!

**Pull request merged**